salt/modules/opsgenie.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Can you change these doc strings to use single quotes instead of double? I know that is picky, but that's what we like here at Salt. :)